tasklist/els-schema/src/main/java/io/camunda/tasklist/store/opensearch/FormStoreOpenSearch.java

```diff
@@ -105,6 +109,7 @@ public Optional<FormIdView> getFormByKey(final String formKey) {
       }
     } catch (final OpenSearchException e) {
       if (e.response().status() == HttpURLConnection.HTTP_NOT_FOUND) {
+        LOG.debug("Form with key {} not found, opensearch related exception {}", formKey, e);
```

Comment:
```suggestion
        LOG.debug("Form with key {} not found.", formKey, e);
```
will log the full stacktrace, instead of only the message